src/File/FileFinder.php

```diff
@@ -4,6 +4,7 @@
 
 namespace PhpAT\File;
 
+use SplFileInfo;
```

Comment:
I guess this is a matter of taste but I don't usually declare core classes here, so better have the same approach everywhere. Can you let them like before please?